master/pkg/actor/ref.go

```diff
@@ -204,7 +206,11 @@ func (r *Ref) deleteChild(address Address) {
 func (r *Ref) processMessage() bool {
 	ctx := r.inbox.get()
 
-	r.log.Tracef("get %T, inbox length: %v", ctx.message, r.inbox.len())
+	msgType := fmt.Sprintf("%T", ctx.message)
+	r.log.Tracef("get %s, inbox length: %v", msgType, r.inbox.len())
+	promLabels := []string{ctx.sender.address.path, ctx.recipient.address.path, msgType}
+	defer r.time(promLabels...)()
+	defer r.recordErr(r.err, promLabels...)
```

Comment:
shouldn't `recordErr` also check the error in a returned func, like `time`?

```suggestion
	defer r.recordErr(r.err, promLabels...)()
```

Reply:
yeah good catch